3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,129 @@
+/** Smart-ease - Goes Global! **/
+
+function getFullname(Gender, firstName, surName, useFormalName) {
+  let fullName = "";
+
+  if (Gender === "female") {
+    if (useFormalName === true) {
+      fullName = `Lady ${firstName} ${surName}`;
+    } else {
+      fullName = `${firstName} ${surName}`;
+    }
+  } else {
+    if (useFormalName === true) {
+      fullName = `Lord ${firstName} ${surName}`;
+    } else {
+      fullName = `${firstName} ${surName}`;
+    }
+  }
+
+  return fullName;
+}
+
+const fullname3 = getFullname("female", "Banjamin", "Hughes", false);
+const fullname4 = getFullname("female", "Banjamin", "Hughes", true);
+const fullname1 = getFullname("male", "Banjamin", "Hughes", false);
+const fullname2 = getFullname("male", "Banjamin", "Hughes", true);
+
+console.log(fullname2);
+console.log(fullname1);
+console.log(fullname3);
+console.log(fullname4);
+
+/** Event application **/
+
+function getEventWeekday(daysFromToday) {
+  const days = [
```

Comment:
I noticed a small issue in the `getEventWeekday` function where `getDate()` is used to determine the day of the week. However, `getDate()` returns the day of the month, which may lead to incorrect results.

To correctly determine the day of the week, the function should use `getDay()`. I recommend updating the code as follows:

```javascript
function getEventWeekday(daysFromToday) {
  const days = [
    "Sunday",
    "Monday",
    "Tuesday",
    "Wednesday",
    "Thursday",
    "Friday",
    "Saturday",
  ];
  const today = new Date();
  const futureDate = new Date(today);
  
  // Add the specified number of days to the current date
  futureDate.setDate(today.getDate() + daysFromToday);
  
  // Get the day of the week using getDay()
  return days[futureDate.getDay()];
}

const getEventday = getEventWeekday(9);

console.log(getEventday);